__test__/routes/getClient.test.js

```diff
@@ -0,0 +1,61 @@
+const superTest = require('supertest');
+
+const app = require('../../server/app');
+
+const request = superTest(app);
+
+const dbConnection = require('../../server/database/dbConnection');
+const buildDB = require('../../server/database/index');
+
+describe('post request to /signup', () => {
+  beforeAll(() => buildDB);
+  afterAll(() => dbConnection.close());
```

Comment:
for client routes I prefer to use login instead of signup , after accepting it you can edit these